templates/rust-starter/host/src/main.rs

```diff
@@ -22,7 +22,7 @@ fn main() {
     // ExecutorEnvBuilder::build().
 
     // For example:
-    let input: u32 = 15*2^27 + 1;
+    let input: u32 = 15 * u32::pow(2, 27) + 1;
```

Comment:
Does this fail to compile? If so, I wonder why CI didn't catch it

Reply:
Compiles just fine! It just is `2`, not `2013265921`, because `^` is bitwise xor rather than pow

Reply:
Haha, I see, good call!!